src/printer.js

```diff
@@ -6,14 +8,38 @@ const {
   join,
   line,
   softline
-} = require('prettier').doc.builders;
+} = prettier.doc.builders;
+const { isNextLineEmptyAfterIndex } = prettier.util;
```

Comment:
Yeah, I think it's a matter of personal preference, I tend to avoid requiring the same module twice. But the destructuring option is good, I'll change it to that.